app/components/App.jsx

```diff
@@ -134,6 +134,34 @@ class App extends Component {
     this.setState(state => ({ hamburgerMenuIsOpen: !state.hamburgerMenuIsOpen }));
   }
 
+  getTitleMeta() {
+    if (isSFServiceGuideSite()) {
+      return (
+        <>
+          <title>SF Service Guide</title>
+          <meta property="og:url" content="https://sfserviceguide.org" />
+          <meta property="og:title" content="SF Service Guide" />
+        </>
+      );
+    } else if (isSFFamiliesSite()) {
+      return (
+        <>
+          <title>SF Families</title>
+          <meta property="og:url" content="https://sffamilies.sfserviceguide.org/" />
+          <meta property="og:title" content="SF Families" />
+        </>
+      );
+    } else {
+      return (
+        <>
+          <title>AskDarcel</title>
+          <meta property="og:url" content="https://askdarcel.org" />
+          <meta property="og:title" content="AskDarcel" />
+        </>
+      );
+    }
```

Comment:
I wonder if we could fold this all into a single, unconditional function that calls other functions to get the correct value for the properties. Since we already have a `getSiteTitle()` function, that could be used for both the `<title>` tag and the `<meta property="og:title">` tag. It would hopefully be easy to create a similar function for the site URL.

Reply:
that's a good idea!